src/main/app/src/app/bag/bag-locatie/bag-locatie.component.ts

```diff
@@ -88,7 +89,7 @@ export class BagLocatieComponent implements OnInit, AfterViewInit, OnChanges {
     const kaartSource = new source.WMTS({
       layer: "standaard",
       format: "image/png",
-      url: "https://service.pdok.nl/brt/achtergrondkaart/wmts/v2_0",
+      url: environment.ACHTERGRONDKAART_API_URL,
```

Comment:
Maybe consider translating this to English? `BACKGROUND_MAP_API_URL` or something?